shopfloor/actions/data.py

```diff
@@ -309,9 +309,17 @@ def _product_parser(self):
             ("seller_ids:supplier_code", self._product_supplier_code),
         ]
 
-    def _product_packaging(self, rec, field):
+    def _product_packaging(self, rec, field, apply_excluded=True):
```

Comment:
is `apply_excluded` used anywhere?

Reply:
No, I added it here in case we ever need to bypass this new logic.